js/state/siteSettings.js

```diff
@@ -68,6 +70,8 @@ module.exports.activeSettings = (siteSettings, appState, appConfig) => {
     })()
   })
 
+  setUserPref('safebrowsing.enabled', settings['safeBrowsing'] && settings['safeBrowsingAll'])
```

Comment:
i think this should be called in the reducers for when settings/siteSettings changed, instead of every time that `activeSettings` is called.

see  `userPrefsUpdateTrigger` in `js/state/contentSettings.js` for an example